### PR TITLE
Fix crash when reversing direction

### PR DESCRIPTION
**Pull request by awalker4**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-game/__pycache__
\ No newline at end of file
+game/__pycache__
+env/
```
Comment (owalk) [inline .gitignore:2]: why add env/

**Review by owalk — CHANGES_REQUESTED** [review]
The .gitignore change is out of the commit scope, can you add another commit

**Comment by awalker4** [comment]
The env directory is for setting up a python [virtualenv](https://pythonbasics.org/virtualenv/) - technically it can be named anything but I've always seen env as convention.

**Comment by owalk** [comment]
I changed my comment when I saw it was in the .gitignore, can you move it to another commit

**Review by owalk — APPROVED** [review]

**Pull request merged**